source/include/platform/acmacosx.h

```diff
@@ -119,6 +119,7 @@
 #include "aclinux.h"
 
 #ifdef __APPLE__
+#undef sem_destroy
 #include <semaphore.h>
 #define sem_destroy         sem_close
```

Comment:
Where is the original position of this line?
Could we move semaphore.h inclusion prior than that line?

Thanks
-Lv